### PR TITLE
add mentors img

### PR DESCRIPTION
**Pull request by kol4k**

### CODE REVIEW

**Comment by nzec** [comment]
@kol4k Use the .yml files in the _data folder to add mentors/student/projects.
Also force push a commit (by adding the arguement -f). This makes sure that you have only one commit.
And also squash the commits. Here is a quick read on how to do that : https://git.io/vb3tC

**Comment by kol4k** [comment]
ok, done squash commit https://github.com/fossasia/gci17.fossasia.org/pull/106/commits/ea4c829a64a30d69de5ce78ead07dbb9d27a079d

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 _site/
 .sass-cache/
+e --interactive HEAD~2
```
Comment (shubham-padia) [inline .gitignore]: What is this for? Please remove this

**Review by shubham-padia — CHANGES_REQUESTED** [review]

**Comment by shubham-padia** [comment]
Good job @kol4k please squash your commits from now on

**Pull request merged**